bwctest.sh

```diff
@@ -80,7 +79,7 @@ done
 if [ -z "$OPENSEARCH" ]; then
     IFS='/' read -ra SLASH_ARR <<< "$DASHBOARDS"
     # Expected to be opensearch-x.y.z-platform-arch.tar.gz
-    TARBALL="${SLASH_ARR[12]}"
+    [[ "$DASHBOARDS" == *"Playground"* ]] && TARBALL="${SLASH_ARR[14]}" || TARBALL="${SLASH_ARR[13]}"
```

Comment:
Where do we define `"Playground"`?

Reply:
Good question, from the domain knowledge of the build repo. Since this script should mainly be used by the build repo side do this ok to keep in?

Reply:
Okay, could we put a reference for where it exists so anyone who looks at/updates this code has context?

Reply:
Added comment. It won't be apparent as much as it is current SOP for sandbox testing.